core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1399,6 +1399,17 @@ private CarbonCommonConstants() {
 
   public static final String CARBON_PUSH_ROW_FILTERS_FOR_VECTOR_DEFAULT = "false";
 
+  /**
+   * max driver threads used for block pruning [1 to 4 threads]
+   */
+  @CarbonProperty public static final String CARBON_MAX_DRIVER_THREADS_FOR_BLOCK_PRUNING =
+      "carbon.max.driver.threads.for.block.pruning";
```

Comment:
I think it's better to use the name
`carbon.query.pruning.parallelism.driver`

Reply:
I have another non-default datamap PR. I will check about this. I feel this name also OK